### PR TITLE
fix(rtcstats): Fix rtcstats poll interval typo and increase default value

### PR DESCRIPTION
**Pull request by nils-ohlmeier**
This PR fixes two typos in the poll interval variable names.

Additionally it increases the default value from 1s to 10s to avoid frequent polling if the default fallback value is used.

### CODE REVIEW

**Review by saghul — APPROVED** [review]

File: config.js
Comment (gpolitis) [inline config.js:971]: Maybe update the default value here as well?

Comment (nils-ohlmeier) [reply config.js:971]: Argh. Sorry. GitHub website showed me everything green. So I pressed the merge button. And only after the merge it showed me your review comment. I should have reloaded the page before merging.

**Comment by jqdrqgnq** [comment]
The comment is stale now (still says 1000ms).

**Comment by damencho** [comment]
It got fixed in the next PR :) https://github.com/jitsi/jitsi-meet/pull/12031

**Comment by jqdrqgnq** [comment]
sorry, the comment in react/features/rtcstats/middleware.js, i think it's still wrong there.

**Comment by damencho** [comment]
Aaa, that one :) yep, you are so right. @nils-ohlmeier can you fix it in some future commit :) Thanks
We should not put those values all around like that :) it's hard to do a simple change correctly :)

**Comment by nils-ohlmeier** [comment]
Thank you @jqdrqgnq for pointing this out. I opened PR #12036 to fix this (hopefully) last one.

**Pull request merged**
